File: Editor/src/main/java/org/turbomedia/turboedit/editor/panes/preferences/PreferencesPane.java
package org.turbomedia.turboedit.editor.panes.preferences;

import javafx.scene.control.TabPane;

public class PreferencesPane extends TabPane {
    public PreferencesPane() {
        setTabClosingPolicy(TabClosingPolicy.UNAVAILABLE);

        getTabs().add(new GeneralTab());
        getTabs().add(new RenderingTab());
        getTabs().add(new PerformanceTab());
        getTabs().add(new SocialMediaTab());
    }
}

File: Editor/src/main/kotlin/turboedit/editor/panes/preferences/PreferencesPane.kt
package turboedit.editor.panes.preferences

import javafx.scene.control.TabPane

class PreferencesPane : TabPane() {
    init {
        tabClosingPolicy = TabClosingPolicy.UNAVAILABLE

        tabs.add(GeneralTab())
        tabs.add(RenderingTab())
        tabs.add(PerformanceTab())
        tabs.add(SocialMediaTab())
    }
}
